modules/postgresql/collector.go

```diff
@@ -0,0 +1,49 @@
+package postgresql
+
+import (
+	"github.com/NETWAYS/support-collector/pkg/collection"
+	"os"
+)
+
+const ModuleName = "postgresql"
+
+var relevantPaths = []string{
+	"/etc/postgresql",
+	"/var/lib/pgsql",
+}
+
+var files = []string{
+	"/etc/postgresql*",
+	"/var/lib/pgsql*",
```

Comment:
On Debian / Ubuntu this would be `/var/lib/postgresql`

But we should not really retrieve the database content.

```suggestion
	"/var/lib/pgsql/data/*.conf", // RedHat based systems, where the configuration is found
```